homeassistant/components/edl21/__init__.py

```diff
@@ -1 +1,22 @@
 """The edl21 component."""
+import asyncio
+import logging
+
+from homeassistant import config_entries, core
+
+from .sensor import DOMAIN
+
+
+async def async_setup_entry(
+    hass: core.HomeAssistant, entry: config_entries.ConfigEntry
+) -> bool:
+    """Set up platform from a ConfigEntry."""
+    hass.data.setdefault(DOMAIN, {})
+    hass_data = dict(entry.data)
+    hass.data[DOMAIN] = hass_data
+
+    # Forward the setup to the sensor platform.
+    hass.async_create_task(
+        hass.config_entries.async_forward_entry_setup(entry, "sensor")
+    )
```

Comment:
Use:

`hass.config_entries.async_setup_platforms(entry, PLATFORMS)`

instead

Reply:
Unresolved this one, as it has not been resolved.

Reply:
So Platform.SENSOR is wrong? Where do I get the constant PLATFORMS?